### PR TITLE
Moving runtime bindings into their own crate

### PR DESCRIPTION
**Pull request by brianjjones**
The runtime bindings have no use for the the build-dependencies of tensorflow-sys, nor the build.rs. So to avoid pulling in those dependencies, and hopefully to keep things cleaner, I'd like to move them into their own crate.

### CODE REVIEW

**Comment by brianjjones** [comment]
Hi @adamcrume , could I get some feedback on this when you get a chance? Thanks.

File: tensorflow-sys-runtime/generate_bindgen_rs.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+echo ${cmd}
+${cmd}
+
+bindgen_options_runtime_types="--allowlist-type TF_.+ --blocklist-function .+ --size_t-is-usize --default-enum-style=rust --generate-inline-functions"
+cmd="bindgen ${bindgen_options_runtime_types} ${include_dir}/tensorflow/c/c_api.h --output src/runtime_linking/types.rs --  -I ${include_dir}"
+echo ${cmd}
+${cmd}
+
+echo "link! {\n$(cat src/runtime_linking/c_api.rs)" > src/runtime_linking/c_api.rs
+echo } >> src/runtime_linking/c_api.rs
```
Comment (adamcrume) [inline tensorflow-sys-runtime/generate_bindgen_rs.sh:23]: All the runtime bits should be removed from tensorflow-sys/generate_bindgen_rs.sh.

**Review by adamcrume — CHANGES_REQUESTED** [review]
RELEASING.md needs to be updated.

**Comment by adamcrume** [comment]
Sorry for the delay!

**Comment by brianjjones** [comment]
@adamcrume Thanks for the review. I've updated the PR with the requested changes, let me know if there's anything else.

**Comment by adamcrume** [comment]
Thanks!

**Comment by brianjjones** [comment]
@adamcrume Great, thanks for that. Any idea when I can expect a new version to be pushed to crates.io?

**Pull request merged**